api/tests/command_test.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management import call_command
```

Comment:
I've been running this command with `python manage.py test` and I believe it is getting run. I can make the test fail by removing files or messing with the ingest command. 

I think it would be possible to run a slimmer data set in the future. While we're adding data sets, I'd like to keep testing the entire import.